src/lib/Components/Inbox/Conversations/Previews/ImagePreview.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import { TouchableHighlight } from "react-native"
+
+import styled from "styled-components/native"
+import colors from "../../../../../data/colors"
+import fonts from "../../../../../data/fonts"
+import OpaqueImageView from "../../../OpaqueImageView"
+
+const Container = styled.View`
+  borderWidth: 1
```

Comment:
Make sure to add stylefmt too so that it's automated: https://github.com/artsy/emission/issues/599